app/views/donations/edit.html.erb

```diff
@@ -9,8 +9,8 @@
       <i class="fa fa-dashboard"></i> Home
     <% end %>
     </li>
-    <li><%= link_to "Donations", (donations_path) %></li>  
-    <li class="active"> Editing <%= @donation.source %> on <%= @donation.created_at.strftime("%B %-d %Y") %></li>
+    <li><%= link_to "Donations", (donations_path) %></li>
+    <li class="active"> Editing <%= @donation.source %> on <%= @donation.created_at.to_formatted_s(:short) %></li>
```

Comment:
Why are you using `to_formatted_s(:short)` instead of `to_s(:distribution_date)`?

Reply:
confusion be my part, was trying to_formatted and it slipped through my changes